src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OperatorOverloadsHaveNamedAlternates.cs

```diff
@@ -1,69 +1,262 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System;
+using System.Collections.Generic;
 using System.Collections.Immutable;
+using System.Linq;
 using Analyzer.Utilities;
-using Microsoft.CodeAnalysis.Diagnostics;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
 
 namespace Microsoft.ApiDesignGuidelines.Analyzers
-{                   
+{
```

Comment:
You can make this type sealed and delete the two derived type since they don't have any functionality. You'll need to add a `[DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic]` attribute to this type